src/main/io/ledstrip.c

```diff
@@ -568,27 +567,27 @@ static void applyLedBatteryLayer(bool updateNow, uint32_t *timer)
     static bool flash = false;
 
     int state;
-    int timeOffset = 1;
+    int frequency = 1;
```

Comment:
Yes it would. But at least a simple rename shows that it is a frequency and hence using 0 will cause problems.